spec.md

```diff
@@ -1997,6 +2006,133 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListSnapshots` operation again with an empty `starting_token`. |
 
 
+#### `ListSnapshotDeltas`
+
+**ALPHA FEATURE**
+
+A Controller Plugin MUST implement this RPC call if it has the
+`LIST_SNAPSHOT_DELTAS` capability. The Plugin SHALL return the
+information about the deltas between the two provided snapshots on the storage
+system within the given parameters regardless of how they were created.
+`ListSnapshotDeltas` SHALL NOT list deltas of snapshots that are being created
+but have not been committed to the storage system. If snapshots are created
+and/or deleted while the CO is concurrently paging through `ListSnapshotDeltas`
+results, then it is possible that the CO MAY either witness duplicate results in
+the list, not witness existing deltas, or both. The CO SHALL NOT expect a
+consistent "view" of all snapshot deltas when paging through the snapshot list
+via multiple calls to `ListSnapshotDeltas`.
+
+A Controller Plugin MUST NOT attempt to include any raw data blocks in the RPC
+response to ensure data plane operations remain outside the scope of CSI. The
+response payload MUST contain enough block location metadata for the end users
+to retrieve the raw data blocks. The end users are responsbile for devising
+their own transport and retrieval mechanism to access the raw data blocks.
+
+```protobuf
+message ListSnapshotDeltasRequest {
+  option (alpha_message) = true;
+
+  // The ID of the base snapshot handle to use for comparison. If
+  // not specified, return all changed blocks up to the target
+  // specified by snapshot_target. This field is OPTIONAL.
```

Comment:
`snapshot_target` is not part of this message, I think you mean `to_snapshot_id`.